packages/adapter-pg/vitest.config.mts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    globals: true,
```

Comment:
fwiw in other packages where we are already using vitest we import `test`, `describe`, `expect` etc explicitly, but I don't have a strong opinion and we can easily change it to be consistent later, so I think it's fine

Reply:
Thank you. I was thinking the same, at first make it work then improve.